### PR TITLE
Update "TH Installation on Raspberry Pi" section

### PR DESCRIPTION
**Pull request by hiltonlima**
Removed installation information using an image

https://github.com/project-chip/certification-tool/blob/update_th_instalation_steps/docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

### CODE REVIEW

**Review by antonio-amjr — APPROVED** [review]

**Review by rquidute — APPROVED** [review]

**Review by rquidute — APPROVED** [review]

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 1]
```diff
@@ -164,44 +165,57 @@ If the DUT is a controlee (e.g., light bulb), the TH spins a reference commissio
 
 If the DUT is a commissioner/controller, the Test TH spins an example accessory that is shipped with the SDK and uses that for the DUT to provision, control and run certification tests.
 
-Refer to <<bringing-up-of-matter-node-dut-for-certification-testing, Section 5, Bringing Up of Matter Node (DUT) for Certification Testing>> to bring up the DUT and then proceed with device testing by referring to <<test-configuration, Section 8, Test Configuration>>.
+Refer to <<bringing-up-of-matter-node-dut-for-certification-testing, Section 5, Bringing Up of Matter Node (DUT) for Certification Testing>> to bring up the DUT and then proceed with device testing by referring to <<test-configuration, Section 7, Test Configuration>>.
```
Comment (ccruzagralopes) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:168]: Should this be changed?

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc [hunk 2]
```diff
@@ -164,44 +165,57 @@ If the DUT is a controlee (e.g., light bulb), the TH spins a reference commissio
...
+** `$cd certification-tool`
+* Initialize the submodules: 
+** `$git submodule update --init --recursive`
+* Install/configure the TH dependencies: 
+** `$./scripts/pi-setup/auto-install.sh`
+** At the end of the script, select option 1 to restart the RaspberryPi.
+. Wait about 10 minutes. 
 . Using the _ifconfig_ command, obtain the IP address of the Raspberry Pi. The same IP address will be used to launch the TH user interface on the user's system using the browser.
-. Proceed with test configuration and execution (refer to <<test-configuration, Section 8, Test Configuration>> and <<test-case-execution, Section 9, Test Case Execution>> respectively).
+. Proceed with test configuration and execution (refer to <<test-configuration, Section 7, Test Configuration>> and <<test-case-execution, Section 8, Test Case Execution>> respectively).
```
Comment (ccruzagralopes) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:218]: Should this be changed?

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc (hunk 1 above)
Comment (hiltonlima) [reply docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:168]: I think it is correct.
image

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc (hunk 2 above)
Comment (hiltonlima) [reply docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:218]: ??
image

File: docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc (hunk 1 above)
Comment (hiltonlima) [inline docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc:168]:
```suggestion
Refer to <<bringing-up-of-matter-node-dut-for-certification-testing, Section 5, Bringing Up of Matter Node (DUT) for Certification Testing>> to bring up the DUT and then proceed with device testing by referring to <<test-configuration, Section 8, Test Configuration>>.
```

**Review by ccruzagralopes — APPROVED** [review]

**Pull request merged**